test/Interop/Cxx/union/Inputs/anonymous-union-partly-invalid.h

```diff
@@ -0,0 +1,16 @@
+@class C;
+@interface C
+{}
+@end
+
+struct S {
+  union {
+    C *t;
```

Comment:
Did we ever figure out _why_ the compiler thinks this is invalid?

Reply:
I believe because it thinks the destructor is deleted at 

https://github.com/apple/swift/blob/13449004a069f7d39d841c675c07d7d6debf0f4a/lib/ClangImporter/ImportDecl.cpp#L3352

in `isCxxRecordImportable`